### PR TITLE
docs: add credly badges page

### PR DESCRIPTION
**Pull request by edodusi**

### CODE REVIEW

File: content/tools-and-policies/credly-badges-assignment.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/*
+Description: How to obtain and assign Credly badges
+Sort: 30
+*/
+
+SparkFabrik uses [Credly](https://www.credly.com/organizations/sparkfabrik) to issue digital badges to team members who have demonstrated specific achievements and made significant contributions to the company. These badges are a way to recognize and reward your dedication to our values and mission.
```
[the remark this answers is not in the document]
Comment (stickgrinder) [reply content/tools-and-policies/credly-badges-assignment.md:6]: Comment to my comment... this goes basically through the whole document. It's not a big deal.

If you feel like adapting the registry to an impersonal audience is not too much effort, I would personally prefer it, considering this is a policy.  If not the current version can do and you can ignore this comment.

File: content/tools-and-policies/credly-badges-assignment.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+At SparkFabrik, we believe it is important to recognize and reward your dedication and commitment above and beyond your day-to-day work. We want to celebrate your accomplishments and contributions, and make you an inspiration to your colleagues and the community.
+
+This is unrelated to your salary and career path, which are managed through other processes and reviews within the company; no one is penalized for not having badges, and no one is ever forced to get them.
+
+We just want to give you a way to show off your achievements and be proud of your contributions.
+
+## Badges issued by SparkFabrik
+
+SparkFabrik issues the following Credly badges:
```
Comment (stickgrinder) [inline content/tools-and-policies/credly-badges-assignment.md:34]: Are those linkable to the Credly catalog? Maybe we can link the titles.

File: docker-compose.yml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 version: "3"
 services:
   documentation:
+    ports:
+      - "127.0.0.1:80:80"
```
Comment (stickgrinder) [inline docker-compose.yml:5]: Why this? We should stick to the local domain model, as for any other project.

If there is a problem with that, it's better to open another PR with an explanation. Thanks

File: content/tools-and-policies/credly-badges-assignment.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+## What are Credly badges?
+
+Credly badges are digital representations of a skill, achievement, or experience that can be shared online. They are a way to showcase your accomplishments and skills to the world.
+
+## How to obtain a badge
+
+To obtain a Credly badge, you must meet the criteria for that badge. For every badge, there are specific requirements that you must fulfill to be eligible to receive it, and these requirements are outlined in the badge description that is available on the Credly platform.
+
+We will review your contributions and achievements periodically and issue badges to those who have met the criteria for each badge. If you believe you have met the criteria for a badge but have not received it, please reach out to your team lead or the person responsible for issuing badges.
```
Comment (stickgrinder) [inline content/tools-and-policies/credly-badges-assignment.md:16]:
```suggestion
The company reviews contributions and achievements periodically and issue badges to those who have met the criteria for each badge. If you believe you have met the criteria for a badge but have not received it, please reach out to your team leader, or to an HR or DevRel representative.
```

Comment (stickgrinder) [reply content/tools-and-policies/credly-badges-assignment.md:16]: Specifying the point of contacts makes this more attainable.

**Review by stickgrinder — CHANGES_REQUESTED** [review]
I like this a lot, just left a couple comments, but overall that's very good.

I've put in "Request changes" to address the point in the docker-compose.yml. The rest, consider it as a comment.

File: content/tools-and-policies/credly-badges-assignment.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (edodusi) [reply content/tools-and-policies/credly-badges-assignment.md:6]: @stickgrinder I followed the practice I see in other pages (i.e. "Attending Conferences"), but if you want I can change it throughout the file

[the remark this answers is not in the document]
Comment (stickgrinder) [reply content/tools-and-policies/credly-badges-assignment.md:6]: @edodusi sorry, I didn't check. Leave it untouched then, we can have a pass on the whole section later for consistency. Thanks

File: content/tools-and-policies/credly-badges-assignment.md (hunk 2 above)
Comment (edodusi) [reply content/tools-and-policies/credly-badges-assignment.md:34]: ok

File: docker-compose.yml (hunk 3 above)
Comment (edodusi) [reply docker-compose.yml:5]: without this the container is not reachable on windows. I can remove it now.

**Comment by edodusi** [comment]
@stickgrinder comments fixed

File: docker-compose.yml (hunk 3 above)
Comment (stickgrinder) [reply docker-compose.yml]: Ah, that's the case. OK, please remove it from this PR and let's address this in a different issue. I'm pretty sure we can solve with dinghy somehow.
Thanks

**Review by stickgrinder — APPROVED** [review]

**Comment by stickgrinder** [comment]
Merged, thanks!

**Pull request merged**